src/main/java/com/epam/test_generator/entities/api/JiraSuitAndCaseTrait.java

```diff
@@ -0,0 +1,13 @@
+package com.epam.test_generator.entities.api;
+
+import java.util.Objects;
+
+public interface JiraSuitAndCaseTrait {
+
+    String getJiraKey();
+
+    default boolean isNotRemoved() {
```

Comment:
Image that you need to check if the item is removed: ` if (!isNotRemoved())`. A little confusing, right? Also it is not clear from the name of this method what is removed and from where.